poetry.lock

```diff
@@ -168,105 +186,98 @@ files = [
 
 [[package]]
 name = "datadog"
-version = "0.51.0"
+version = "0.52.1"
 description = "The Datadog Python library"
 optional = false
 python-versions = "!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*,!=3.6.*,>=2.7"
 groups = ["main"]
 files = [
-    {file = "datadog-0.51.0-py2.py3-none-any.whl", hash = "sha256:a9764f091c96af4e0996d4400b168fc5fba380f911d6d672c9dcd4773e29ea3f"},
-    {file = "datadog-0.51.0.tar.gz", hash = "sha256:3279534f831ae0b4ae2d8ce42ef038b4ab38e667d7ed6ff7437982d7a0cf5250"},
+    {file = "datadog-0.52.1-py2.py3-none-any.whl", hash = "sha256:b8c92cd761618ee062f114171067e4c400d48c9f0dad16cb285042439d9d5d4e"},
+    {file = "datadog-0.52.1.tar.gz", hash = "sha256:44c6deb563c4522dba206fba2e2bb93d3b04113c40191851ba3a241d82b5fd0b"},
 ]
 
 [package.dependencies]
 requests = ">=2.6.0"
 
 [[package]]
 name = "ddtrace"
-version = "2.20.0"
+version = "3.11.2"
```

Comment:
I'm not sure we should make this change just yet. This would mean that any customers currently relying on v2 of ddtrace would have to update all of their instrumentation code.

Reply:
It's v3 in `pyproject.toml`. Does this mean we are already requiring customers to use v3?
https://github.com/DataDog/datadog-lambda-python/blob/50d893a402a9ef28d958457a0293b6ab1eb18498/pyproject.toml#L31

Reply:
Oh look at that. Yes, I think you are correct. Good catch.